src/main/webapp/jsp/instructorCourseEnroll.jsp

```diff
@@ -6,117 +6,117 @@
+                <ul>
+                  <li class="moreInfoEmailDetails">Team must be unique within a course. A team cannot be in 2 different sections.</li>
+                  <li class="moreInfoEmailDetails">If you do not have teams in your course, use “N/A” as the team name for all students.</li>
+                </ul>
+              </div>
             </li>
-            <li><span class="moreInfoPointTitle">Enrolling without spreadsheets</span>
-                <div class="moreInfoPointDetails">
-                    The alternative is to type student data in the text box, using the pipe symbol (also called the vertical bar,
-                        not to be confused with upper case i or lower case L) to separate values.
-                    <br> Here is an example.
-                    <br><br>
-                    <div id="moreInfoEnrollWOSpreadSheetEg">
-                        <samp>
-                            <span class="enrollLines"><b>Section   |   Team   |   Name   |   Email   |   Comments</b></span>
-                            <br><span class="enrollLines">Tut Group 1   |   Team 1   |   Tom Jacobs  |  tom@email.com</span>
-                            <br><span class="enrollLines">Tut Group 1   |   Team 1  |   Jean Wong   |   jean@email.com   |   Exchange Student</span>
-                            <br><span class="enrollLines">Tut Group 1   |   Team 2   |   Jack Wayne  |  jack@email.com</span>
-                            <br><span class="enrollLines">Tut Group 2   |   Team 3   |   Thora Parker  |  thora@email.com</span>
-                        </samp>
-                    </div>
-                </div>
+            <li class="moreInfoColumnInfo"><samp>Name</samp> [Compulsory]: Student name</li>
+            <li class="moreInfoColumnInfo"><samp>Email</samp> [Compulsory]: The email address used to contact the student.<br>
+              <div class="moreInfoPointDetails">
+                <ul>
+                  <li class="moreInfoEmailDetails">This need not be a Gmail address.</li>
+                  <li class="moreInfoEmailDetails">It should be unique for each student.
+                      If two students are given the same email, they will be considered the same student.</li>
+                </ul>
+              </div>
             </li>
-        </ul>
-    </div>
+            <li class="moreInfoColumnInfo"><samp>Comments</samp> [Optional]: Any other information you want to record about a student.</li>
+          </ul>
+        </div>
+      </li>
+      <li><span class="moreInfoPointTitle">Mass editing enrolled students</span>
+        <div class="moreInfoPointDetails">
+          <ul>
+            <li class="moreInfoMassEditInfo">The text box above can be used for mass-editing details (except email address) of students already enrolled.
+                                 To edit, simply enroll students using the updated data and existing data will be updated accordingly.</li>
```

Comment:
@sujeet14108 if you encounter stuffs like this, you should fix it. Since this PR is specifically touching indentation and practically every line of the JSP files, we'll also be looking for this kind of occurrence.